db/errors.go

```diff
@@ -59,12 +60,24 @@ const (
 	errIndexDescriptionHasNoFields    string = "index description has no fields"
 	errIndexDescHasNonExistingField   string = "index description has non existing field"
 	errFieldOrAliasToFieldNotExist    string = "The given field or alias to field does not exist"
+	errCreateFile                     string = "failed to create the file"
```

Comment:
nitpick: `the` in these error messages feels quite odd to me. It might also be misleading if the filepath is not noted in the error (e.g. in case error was from a internal-file and not the user-specified path).

Reply:
✅